examples/models/transform_jitter.py

```diff
@@ -1,3 +1,18 @@
+"""
+==========================================
+Jittered Scatter Plot with Bokeh and NumPy
+==========================================
+
+This example demonstrates how to add jitter to a scatter plot using Bokeh and NumPy.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.models.sources.ColumnDataSource, bokeh.models.markers.Circle, bokeh.models.Jitter, bokeh.models.LabelSet, bokeh.models.widgets.Button, bokeh.models.callbacks.CustomJS
```

Comment:
I think this path needs to be updated:

```suggestion
    :apis: bokeh.plotting.figure.circle, bokeh.models.sources.ColumnDataSource, bokeh.models.Circle, bokeh.models.Jitter, bokeh.models.LabelSet, bokeh.models.Button, bokeh.models.CustomJS
```

Use the reference guide at https://docs.bokeh.org/en/latest/docs/reference.html to check the paths to use here (usually the parts after the `#` in the URL), e.g. `bokeh.models.RadioGroup` from `https://docs.bokeh.org/en/latest/docs/reference/models/widgets/groups.html#bokeh.models.RadioGroup`